src/Templates/Loadouts/Arma3_CIVILIAN/A3_CIVILIAN_Default_Small_Arms.sqf

```diff
@@ -0,0 +1,189 @@
+    "U_C_Commoner_shorts",
+    "U_C_HunterBody_grn",
+    "U_C_Journalist",
+    "U_C_Man_casual_1_F",
+    "U_C_Man_casual_2_F",
+    "U_C_Man_casual_3_F",
+    "U_C_Man_casual_4_F",
+    "U_C_Man_casual_5_F",
+    "U_C_Man_casual_6_F",
+    "U_C_man_sport_1_F",
+    "U_C_man_sport_2_F",
+    "U_C_man_sport_3_F",
+    "U_C_Mechanic_01_F",
+    "U_C_Paramedic_01_F",
+    "U_C_Poloshirt_blue",
+    "U_C_Poloshirt_burgundy",
+    "U_C_Poloshirt_redwhite",
+    "U_C_Poloshirt_salmon",
+    "U_C_Poloshirt_stripped",
+    "U_C_Poloshirt_tricolour",
+    "U_C_Poor_1",
+    "U_C_Poor_2",
+    "U_C_TeeSurfer_shorts_1",
+    "U_C_TeeSurfer_shorts_2",
+    "U_C_WorkerCoveralls",
+    "U_Competitor"
+];
+
+this forceAddUniform selectRandom _uniforms;
+
+private _headgear = [
+	"H_Bandanna_blu",
+    "H_Bandanna_camo",
+    "H_Bandanna_cbr",
+    "H_Bandanna_gry",
+    "H_Bandanna_khk",
+    "H_Bandanna_khk_hs",
+    "H_Bandanna_mcamo",
+    "H_Bandanna_sand",
+    "H_Bandanna_sgg",
+    "H_Bandanna_surfer",
+    "H_Bandanna_surfer_blk",
+    "H_Bandanna_surfer_grn",
+    "H_Watchcap_blk",
+    "H_Watchcap_cbr",
+    "H_Watchcap_camo",
+    "H_Watchcap_khk",
+    "H_Cap_blk",
+    "H_Cap_blk_CMMG",
+    "H_Cap_blk_ION",
+    "H_Cap_blk_Syndikat_F",
+    "H_Cap_blu",
+    "H_Cap_grn",
+    "H_Cap_grn_BI",
+    "H_Cap_grn_Syndikat_F",
+    "H_Cap_khaki_specops_UK",
+    "H_Cap_oli",
+    "H_Cap_oli_Syndikat_F",
+    "H_Cap_Orange_IDAP_F",
+    "H_Cap_press",
+    "H_Cap_red",
+    "H_Cap_surfer",
+    "H_Cap_tan",
+    "H_Cap_tan_Syndikat_F",
+    "H_Cap_usblack",
+	"H_HeadBandage_bloody_F",
+    "H_HeadBandage_clean_F",
+    "H_HeadBandage_stained_F",
+	"H_StrawHat",
+    "H_StrawHat_dark",
+    "H_Hat_Safari_olive_F",
+    "H_Hat_Safari_sand_F"
+];
+
+if (random 5 < 1) then { this addHeadgear selectRandom _headgear;
+};
+
+private _vest = [
+	"V_Pocketed_black_F",
+	"V_Pocketed_coyote_F",
+	"V_Pocketed_olive_F"
+];
+
+if (random 10 < 1) then { this addVest selectRandom _vest;
+};
+
+private _gunsAndAmmo = [
+	["hgun_Pistol_heavy_01_F", 				"11Rnd_45ACP_Mag"],	1,
+	["hgun_ACPC2_F", 						"9Rnd_45ACP_Mag"],	1,
+	["hgun_P07_F", 							"16Rnd_9x21_Mag"],	1,
+	["hgun_Rook40_F", 						"16Rnd_9x21_Mag"],	1,
+	["hgun_Pistol_01_F", 					"10Rnd_9x21_Mag"],	1,
+	["hgun_P07_khk_F", 						"16Rnd_9x21_Mag"],	1
+];
+
+(selectRandomWeighted _gunsAndAmmo) params ["_gun", "_ammo"];
+
+this addVest selectRandom ["V_BandollierB_blk", "V_BandollierB_cbr", "V_BandollierB_rgr", "V_BandollierB_khk", "V_BandollierB_oli"];
+
+this addWeapon _gun;
+this addHandgunItem "acc_flashlight_pistol";
+this addHandgunItem _ammo;
+
+if(random 5 < 1) then {
+	this addGoggles selectRandomWeighted [
+		"G_Spectacles", 		1,
+		"G_Sport_Red", 			1,
+		"G_Squares_Tinted", 	1,
+		"G_Squares", 			1,
+		"G_Spectacles_Tinted", 	1,
+		"G_Shades_Black", 		1,
+		"G_Shades_Blue", 		1
+	];
+};
+
+//====Items====
+for "_i" from 1 to 3 do { this addItemToUniform _ammo };
+
+//====ACE Items====
```

Comment:
We don't add ACE items to loadouts because medical ACE is optional, you should add a check if ACE medical is enabled, otherwise people will see an error.